parts/k8s/kubernetesmasterresourcesvmss.t

```diff
@@ -27,44 +16,33 @@
     {{ if UseManagedIdentity}}
     "dependsOn": 
     [
-      "[concat('Microsoft.Compute/virtualMachineScaleSets/', variables('masterVMNamePrefix'), 'vmss')]",
-      "[concat('Microsoft.Authorization/roleAssignments/', guid(concat('Microsoft.Compute/virtualMachineScaleSets/', variables('masterVMNamePrefix'), 'vmidentity')))]",
+      "[concat('Microsoft.Compute/virtualMachineScaleSets/', variables('masterVMNamePrefix'), 'vmss')]"
+      {{if UserAssignedIDEnabled}}
+      ,"[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities/', variables('userAssignedID'))]"
```

Comment:
`'Microsoft.ManagedIdentity/userAssignedIdentities/', variables('userAssignedID'))`

^ Do they need to be concatted?

Reply:
Also, since they are referenced multiple times in the template. Would it make sense to assign `'Microsoft.ManagedIdentity/userAssignedIdentities/', variables('userAssignedID')` to a variable that can re re-used?